frontends/web/src/api/coins.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { subscribeEndpoint, SubscriptionCallback } from './subscribe';
+
+interface Status {
+    targetHeight: number;
+    tip: number;
+    tipAtInitTime: number;
+    tipHashHex: string;
+}
+
+
+export const subscribeCoinHeaders = (coinCode: string) => (
+    (cb: SubscriptionCallback<Status>) => (
```

Comment:
why do you put the type here instead of above in `subscribeCoinHeaders()`? would be nice to see the whole function type at a glance without looking into the body of the function.

Reply:
it is part of `subscribeCoinHeaders` just on a new line, I requested to put new lines, because I have a hard time reading/understanding when there are multiple arrow functions on the same line.